### PR TITLE
configs: Make riscv/fs_linux work in build/ALL/gem5.opt

### PR DESCRIPTION
**Pull request by rogerchang23424**
Change-Id: If9add7dc5e9c5600f769d27817da41466158942b

### CODE REVIEW

**Comment by rogerchang23424** [comment]
Blocked by #652

File: configs/example/riscv/fs_linux.py
Comment (powerjg) [inline configs/example/riscv/fs_linux.py:143]: No need to make any changes, but since this script uses `common/Options.py` it should probably be moved to "deprecated." 

The maintainers are explicitly not maintaining anything that uses `common/Options.py`. That said, I appreciate the community continuing to make these scripts work.

Comment (rogerchang23424) [reply configs/example/riscv/fs_linux.py:143]: I decide to keep the parameter. In the multi-isa system, the default of cpu_type should be the base type of RISC-V CPU.

**Review by BobbyRBruce — APPROVED** [review]

**Comment by powerjg** [comment]
@BobbyRBruce This needs to be cherry-picked to staging

**Pull request merged**
